plugins/brew/_brew

```diff
@@ -106,4 +111,28 @@ case "$words[1]" in
   upgrade)
     _brew_outdated_formulae
     _wanted outdated_formulae expl 'outdated formulae' compadd -a outdated_formulae ;;
+  services)
+     _arguments -C \
+      '1: :->command' \
+      '2: :->service' && return 0
+
+    local -a commands
+
+    commands=(
+      'clenaup:Get rid of stale services and unused plists'
```

Comment:
Hi. This new autocomplete for brew services is a good new feature. Do you have time to fix the typo to merge the pull request? Cheers